internal/device/outofband/outofband.go

```diff
@@ -166,13 +170,13 @@ func (o *Queryor) biosConfiguration(ctx context.Context, bmc BMCQueryor, asset *
 		// increment get bios configuration query error count metric
 		switch {
 		case strings.Contains(err.Error(), "no compatible System Odata IDs identified"):
-			asset.IncludeError("GetBiosConfigurationError", "redfish_incompatible: no compatible System Odata IDs identified")
+			asset.AppendError(GetBiosConfigError, "redfish_incompatible: no compatible System Odata IDs identified")
```

Comment:
Is `System Odata` a typo?

Reply:
not a typo, Odata is the basis of the Redfish API

Reply:
A comment to that effect (if one doesn’t exist that I missed) would be helpful here.